nixos/modules/installer/cd-dvd/installation-cd-graphical-kde.nix

```diff
@@ -27,47 +18,15 @@ with lib;
     synaptics.enable = true;
```

Comment:
Shouldn't this be `libinput`? I don't think it's a default for plasma5.

Reply:
I agree. I didn't want to make any major functionality changes right now.
Running as non-root is major enough for a single PR and warrants consideration purely on it's own rights.